PEP8 conformance test fails without listing location of failures

We are currently seeing this on master:

```
======================================================================
FAIL: matplotlib.tests.test_coding_standards.test_pep8_conformance
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.6/lib/python2.6/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/virtualenv/python2.6/lib/python2.6/site-packages/matplotlib-1.3.0rc2-py2.6-linux-x86_64.egg/matplotlib/tests/test_coding_standards.py", line 217, in test_pep8_conformance
    assert_equal(result.total_errors, 0, "Found code syntax "
AssertionError: Found code syntax errors (and warnings).
----------------------------------------------------------------------
```

Cc: @pelson

--- a/lib/matplotlib/tests/test_coding_standards.py
+++ b/lib/matplotlib/tests/test_coding_standards.py
@@ -141,21 +141,22 @@ if HAS_PEP8:
             '*/matplotlib/projections/__init__.py',
             '*/matplotlib/projections/geo.py',
             '*/matplotlib/projections/polar.py']
-    
+
         #: A class attribute to store the lines of failing tests.
         _global_deferred_print = []
-    
+
         #: A class attribute to store patterns which have seen exceptions.
         matched_exclusions = set()
-    
+
         def get_file_results(self):
-            # If the file had no errors, return self.file_errors (which will be 0)
+            # If the file had no errors, return self.file_errors
+            # (which will be 0).
             if not self._deferred_print:
                 return self.file_errors
-    
-            # Iterate over all of the patterns, to find a possible exclusion. If we
-            # the filename is to be excluded, go ahead and remove the counts that
-            # self.error added.
+
+            # Iterate over all of the patterns, to find a possible exclusion.
+            # If the filename is to be excluded, go ahead and remove the
+            # counts that self.error added.
             for pattern in self.expected_bad_files:
                 if fnmatch(self.filename, pattern):
                     self.matched_exclusions.add(pattern)
@@ -168,18 +169,17 @@ if HAS_PEP8:
                         self.file_errors -= 1
                         self.total_errors -= 1
                     return self.file_errors
-    
+
             # mirror the content of StandardReport, only storing the output to
             # file rather than printing. This could be a feature request for
             # the PEP8 tool.
             self._deferred_print.sort()
-            for line_number, offset, code, text, doc in self._deferred_print:
+            for line_number, offset, code, text, _ in self._deferred_print:
                 self._global_deferred_print.append(
-                    self._fmt % {
-                    'path': self.filename,
-                    'row': self.line_offset + line_number, 'col': offset + 1,
-                    'code': code, 'text': text,
-                })
+                    self._fmt % {'path': self.filename,
+                                 'row': self.line_offset + line_number,
+                                 'col': offset + 1, 'code': code,
+                                 'text': text})
             return self.file_errors
 
 
@@ -207,9 +207,9 @@ def test_pep8_conformance():
     reporter = pep8style.options.reporter
 
     # Extend the number of PEP8 guidelines which are not checked.
-    pep8style.options.ignore = pep8style.options.ignore + ('E121', 'E122',
-                                    'E123', 'E124', 'E125', 'E126', 'E127',
-                                    'E128')
+    pep8style.options.ignore = (pep8style.options.ignore +
+                                ('E121', 'E122', 'E123', 'E124', 'E125',
+                                 'E126', 'E127', 'E128'))
 
     # Support for egg shared object wrappers, which are not PEP8 compliant,
     # nor part of the matplotlib repository.

--- a/lib/matplotlib/tests/test_table.py
+++ b/lib/matplotlib/tests/test_table.py
@@ -7,12 +7,11 @@ from matplotlib.testing.decorators import image_comparison
                   extensions=['png'],
                   remove_text=True)
 def test_zorder():
-    data = [[  66386,  174296,],
-            [  58230,  381139,]]
+    data = [[66386, 174296],
+            [58230, 381139]]
 
     colLabels = ('Freeze', 'Wind')
     rowLabels = ['%d year' % x for x in (100, 50)]
-
 
     cellText = []
     yoff = np.array([0.0] * len(colLabels))

